l1-contracts/contracts/common/WrappedStandardERC20.sol

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {ERC20PermitUpgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/draft-ERC20PermitUpgradeable.sol";
+import {UpgradeableBeacon} from "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import {ERC1967Upgrade} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Upgrade.sol";
+
+import {IStandardToken} from "../bridge/interfaces/IStandardToken.sol";
+import {EmptyAddress, Unauthorized, NonSequentialVersion, Unimplemented} from "./L1ContractErrors.sol";
+
+/// @author Matter Labs
+/// @custom:security-contact security@matterlabs.dev
+/// @notice The ERC20 token implementation, that is used in the "default" ERC20 bridge. Note, that it does not
+/// support any custom token logic, i.e. rebase tokens' functionality is not supported.
+contract WrappedStandardERC20 is ERC20PermitUpgradeable, IStandardToken, ERC1967Upgrade {
```

Comment:
This is the L2StandardToken just renamed right? Maybe instead of Wrapped it should be Bridged ? ( for me wrapped is weth )